orchard-ws/app/controllers/WorkflowController.scala

```diff
@@ -87,10 +87,26 @@ class WorkflowController @Inject() (
         if resource.maxAttempt <= 0 || resource.maxAttempt > 100
       } yield resource.maxAttempt
 
+    val definedResIds = request.resources.map(_.id)
+    val invalidResourceDefined = for {
```

Comment:
I believe we also wanna capture other invalid requests, like invalid resource type, instance type etc.

Reply:
I originally thought about adding resource compatibility checks, but decided not to include in this PR because it was not clear which component(s) should be responsible for this.

Since orchard has multiple providers (AWS, Falcon), should we let Falcon and AWS system themselves tell us at run time, since they are better to encapsulate **up-to-date** resource catalog/capabilities/compatibilities.  I understand this approach, however, will give the user a slower response.

Reply:
can validate in aws provider?

Reply:
Thanks for the feedback.  I will look into adding AWS activity and resource with a simple metadata layer.

Reply:
I think this works for now